packages/guideline-blocks-settings/tsconfig.json

```diff
@@ -14,6 +14,7 @@
         "sourceMap": true,
         "strict": true,
         "target": "es6",
+        "jsx": "react",
```

Comment:
```suggestion
        "jsx": "react-jsx",
```
- `import React from 'react'` line can be removed
- `import React, { useState } from 'react'` becomes `import { useState } from 'react'`

Reply:
Updated, and removed all react imports.